modules/basegl/src/processors/volumeslicegl.cpp

```diff
@@ -470,7 +478,7 @@ void VolumeSliceGL::renderPositionIndicator() {
 
     utilgl::GlBoolState smooth(GL_LINE_SMOOTH, true);
     utilgl::BlendModeState blend(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
-    utilgl::LineWidthState linewidth(2.5f);
+    //utilgl::LineWidthState linewidth(2.5f); // Doesn't seem to make a difference
```

Comment:
You can remove it. It is not supported in OpenGL anymore.
Use linerenderer instead

Reply:
Perhaps you could mark LineWidthState as deprecated?

Reply:
How can I mark it deprecated? And should I? As Richard says it might be supported for some people.

Reply:
Noone should use it IMHO.
Example:

```
[[deprecated("was declared deprecated. Use `onChange(std::function<void()>)` instead")]]
const BaseCallBack* onChange(T* o, void (T::*m)());
```

Reply:
Yeah, but where do I put "[[deprecated]]"? I tried putting it above this line but it just says "attributes are not allowed here":
![image](https://user-images.githubusercontent.com/8807821/55400959-914c8800-554f-11e9-9336-2ff690946a6d.png)
And I don't want to deprecate SimpleState since it's a template struct used for other stuff as well.

Reply:
Inbetween 
`using PS [[deprecated]] = S*;`
https://en.cppreference.com/w/cpp/language/attributes/deprecated

Reply:
0ffd3b4 this alright?